vhd-notes/AKSWindows/2019/17763.2803.220413.txt

```diff
@@ -0,0 +1,177 @@
+Installed Features
```

Comment:
💡 Please note: this file has missing this information which was consistent in previous release notes: https://github.com/Azure/AKS/blob/master/vhd-notes/AKSWindows/2019/17763.2686.220309.txt#L1-L15 

Not sure if that is a concious decision but just a little fyi. Thanks.

Reply:
That's build metadata related to the Azure DevOps pipeline and hosted agent used to build the image and is not related to the artifact produced, so we intentionally removed that metadata.